firebase/podcasts.js

```diff
@@ -3,9 +3,17 @@ import firebase from 'firebase/app'
 /**
  * Fetch podcasts
  */
+
+var collection = 'podcasts'
+
 export const fetchPodcasts = () =>
   firebase
     .firestore()
-    .collection('podcasts')
+    .collection(collection)
     .get()
-    .then(result => result.docs.map(ref => ({ id: ref.id, ...ref.data() })))
+    .then(result =>
+      result.docs.map(ref => ({
+        id: ref.id,
+        title: ref.data().title
```

Comment:
In this situation, Firestore fetched the entire object. So there is no need to filter object properties by yourself (User can still access those properties in network requests if he really wants to). I suggest you to keep the entire object, so you won't need to refetch missing properties later with an extra request (in the detail view for example)